### ISSUE TITLE
Stochastic gradient descent precision 

### ISSUE THREAD

**Issue by lcf87** [mingyue312/TensorFlowFinance#2]
After changing to stochastic training, the precision dropped by roughly 5%
